app/widgets/GridSearch.js

```diff
@@ -251,7 +251,9 @@ define([
       } else {
         this.addActorLink.style.display = 'none';
       }
-      this._refresh();
+      setTimeout(function () {
+        this._refresh();
+      }.bind(this), 0);
```

Comment:
arrow function kon ook ipv .bind(this), maar niet zeker of alle dojo applicaties dit al ondersteunen (vanaf es6)

Reply:
best geen es6 idd. the old way is met lang.hitch, maar komt op zelfde neer:
  setTimeout(lang.hitch(this, function() {
    this.isStateChanged();
  }), 0);